setup.cfg

```diff
@@ -44,6 +44,10 @@ incremental = True
 check_untyped_defs = True
 warn_unused_ignores = True
 
+[tool:pytest]
+filterwarnings =
+    ignore::DeprecationWarning:docutils.io
```

Comment:
At first, I added this filter into  `_filter_warnings()` in `tests/conftest.py`. But is is not work as intended.
I doubt the function is working.

And I also noticed `PYTHONWARNINGS` is not in passenv of tox.ini. And there is definition of `setenv PYTHONWARNINGS` in tox.ini.
So the definition on `.travis.yml` might have only made noise...